react/package.json

```diff
@@ -8,6 +8,7 @@
   "dependencies": {
     "apollo-cache-inmemory": "^1.1.5",
     "apollo-client": "^2.2.0",
+    "apollo-link-batch-http": "^1.2.1",
```

Comment:
Is this necessary?

Reply:
Sorry, interferences from other features, changed now